dask/dataframe/tests/test_groupby.py

```diff
@@ -2684,3 +2684,43 @@ def test_groupby_iter_fails():
     ddf = dd.from_pandas(df, npartitions=1)
     with pytest.raises(NotImplementedError, match="computing the groups"):
         list(ddf.groupby("A"))
+
+
+def test_groupby_scalar_column():
+    df = pd.DataFrame(
+        [
+            [0, 1, 2],
+            [0, 1, 2],
+            [0, 1, 2],
+            [0, 1, 2],
+        ],
+        columns=[2, 1, 0],
+    )
+    ddf = dd.from_pandas(df, npartitions=2)
+    expected = df.groupby(2).cov()
+    result = ddf.groupby(2).cov()
+    assert_eq(result, expected)
+
+
+@pytest.mark.parametrize(
+    ["op"],
+    [
+        [lambda s: s.agg("sum")],
+        [lambda s: s.cov()],
```

Comment:
I think we don't need to check for `cov()` here. By definition, we take `cov` between two columns (that's why we get the [`cov() missing 1 required positional argument: 'other'`](https://github.com/dask/dask/runs/6338467120?check_suite_focus=true#step:6:21599) error), which is covered by your previous test. :)

```suggestion
```